Library/Homebrew/cmd/list.rb

```diff
@@ -103,8 +103,7 @@ def list
         ls_args << "-t" if args.t?
 
         if !$stdout.tty? && !args.formula?
-          odeprecated "`brew list` to only list formulae", "`brew list --formula`"
-          safe_system "ls", *ls_args, HOMEBREW_CELLAR
+          odisabled "`brew list` to only list formulae", "`brew list --formula`"
```

Comment:
this was only recently `odeprecated` in #8851 and is related to the deprecation of `brew cask` commands, so I've opened #9357 to revert the change to this file so it will remain `odeprecated` for 2.6.0